### PR TITLE
Update data-classification-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Edited prerequisites based on https://opdhsblobprod04.blob.core.windows.net/contents/e0993bf6e82e4fa8ac18b48839521869/2ce5e803079c3427c011efbce08b900a?sv=2018-03-28&sr=b&si=ReadPolicy&sig=KAG1lGSeWNpMVjfNAkfb1pKjGDpk8kuCyiMNPbZE%2BpA%3D&st=2020-08-10T19%3A40%3A49Z&se=2020-08-11T19%3A50%3A49Z

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2568

File: microsoft-365/compliance/data-classification-overview.md [hunk 1]
```diff
@@ -46,8 +46,9 @@ Every account that accesses and uses data classification must have a license ass
 
 - Microsoft 365 (E5)
 - Office 365 (E5)
-- Advanced Compliance (E5) add-on
-- Advanced Threat Intelligence (E5) add-on
+- Microsoft 365 E5 Compliance add on ( Requires Microsoft 365 E3 or Office 365 E3 + EMS E3 )
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-overview.md]:
```suggestion
- Microsoft 365 E5 Compliance add-on (Requires Microsoft 365 E3 or Office 365 E3 + EMS E3)
```

File: microsoft-365/compliance/data-classification-overview.md [hunk 2]
```diff
@@ -46,8 +46,9 @@ Every account that accesses and uses data classification must have a license ass
...
+- Microsoft 365 E5/A5 Info Protection & Governance (Requires AIP Plan 1 or EMS E3 + Exchange Online, SharePoint Online or OneDrive for business)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-overview.md]:
```suggestion
- Microsoft 365 E5/A5 Info Protection & Governance (Requires AIP Plan 1 or EMS E3 + Exchange Online, SharePoint Online, or OneDrive)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Comment by chrfox** [comment]
@msbemba - The link you included at the start of this PR is bad, I don't know where you got your information from.  I have queried the owning PM and will update as necessary

**Comment by msbemba** [comment]
Thank you @chrfox I got it from docs.microsoft.com - https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance

The link that has a PDF file for Detailed Microsoft 365 Compliance Licensing Comparison

Thanks
Sri

**Pull request closed without merging**